variables.tf

```diff
@@ -95,8 +95,14 @@ variable "machine_image" {
   default     = "debian-cloud/debian-9"
 }
 
-variable "machine_network" {
-  description = "The instance network"
+variable "machine_subnetwork" {
+  description = "The instance subnetwork"
   type        = string
   default     = "default"
 }
+
+variable "machine_region" {
+  description = "The instance region"
+  type        = string
+  default     = "default"
```

Comment:
`default` is not a valid zone, please update to `us-west1`

Reply:
updated to us-central1 to match default zone